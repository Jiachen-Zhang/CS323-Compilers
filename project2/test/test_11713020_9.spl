int test_11713020_8()
{
  float f = 1.0;
  char c = 'c';
  return c;
}