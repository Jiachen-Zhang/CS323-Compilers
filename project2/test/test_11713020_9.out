Error type 8 at Line 5: incompatiable return type
